srcpkgs/virt-manager/template

```diff
@@ -22,3 +20,29 @@ post_install() {
 	rm $DESTDIR/usr/share/glib-2.0/schemas/gschemas.compiled \
 		$DESTDIR/usr/share/icons/hicolor/icon-theme.cache
 }
+
+virt-manager-tools_package() {
+	noarch=yes
+	short_desc="Programs to create and clone virtual machines"
+	depends="libosinfo python3-gobject libvirt-python3 python3-urllib3 libxml2-python3 python3-requests"
+	pycompile_dirs="/usr/share/${pkgname}"
```

Comment:
This should be `${sourcepkg}` instead of `${pkgname}` otherwise the name of the subpackage is used and there are no files to compile in this path.

Reply:
done